orchestrator/careplancontributor/service.go

```diff
@@ -50,6 +70,13 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 	mux.HandleFunc("/contrib/cps/fhir/*", s.withSession(func(writer http.ResponseWriter, request *http.Request, _ *user.SessionData) {
 		carePlanServiceProxy.ServeHTTP(writer, request)
 	}))
+	fhirProxy := coolfhir.NewProxy(log.Logger, s.fhirURL, "/contrib/fhir", s.transport)
+	// TODO: Determine auth. CareTeam can be sent in header but this is not ideal, preferably put it in the access token
+	// TODO: Modify this and other URLs to /cpc/* in future change
+	mux.HandleFunc("/contrib/fhir/*", s.withSession(func(writer http.ResponseWriter, request *http.Request, _ *user.SessionData) {
+		// TODO: Is this correct?
+		fhirProxy.ServeHTTP(writer, request)
```

Comment:
This misses actual authorization (does the resource being requested "belong" to the CarePlan being requested), but it might be a good idea to keep that in an upcoming PR. Then we can simultaneously work on authorization (checking Nuts access token, checking CareTeam) and checking the accessed scope (resource v.s. CarePlan).

Reply:
Yeah I wanted to get the review out early to confirm I had correctly set up the Proxy, with Auth coming later. Happy for this to be in a future PR